docs/overview.md

```diff
@@ -9,15 +9,14 @@ BNB Chain, one of the most popular blockchains in the world, dedicates to delive
 
 ## What is Build N Build (BNB) Chain?
 BNB Chain is comprised of:
-1. **[BNB Beacon Chain](learn/beaconIntro.md)** - BNB Chain Governance (Staking, Voting)
-2. **[BNB Smart Chain (BSC)](learn/intro.md)** - EVM compatible, consensus layers, and with hubs to multi-chains
-3. **[BNB Sidechain](BNBSidechain/overview/bs-overview.md)** - Ready-to-use PoS solutions for developing custom blockchains and dApps with existing BSC functionality.
-4. **[ZkBNB](zkbnb/zkbnb-overview.md)** - ZkRollup solution to scale BSC as an super high performance blockchains. 
+1. **[BNB Beacon Chain](learn/beaconIntro.md)** - BNB Beacon Chain(BC) implements a vision of a decentralized exchange (DEX) for digital assets. During the evolution of the dual chain structure, BC plays more like a Beacon Chain that helps to enhance the security of BSC as a staking and governance layer.
```

Comment:
Do we still use such keywords for Beacon Chain `decentralized exchange (DEX) for digital assets` ?